test/directory_input/typescript.d.ts

```diff
@@ -1,27 +1 @@
 /* This file is generated and managed by tsync */
-
-/** Doc comments are preserved too! */
-interface Book {
-  /** Name of the book. */
-  name: string;
-  /** Chapters of the book. */
-  chapters: Array<Chapter>;
-  /**
-   * Reviews of the book
-   * by users.
-   */
-  user_reviews?: Array<string>;
-}
-
-/**
- * Multiple line comments
- * are formatted on
- * separate lines
- */
-interface Chapter {
-  title: string;
-  pages: number;
-}
-
-/** Time in UTC seconds */
-type UTC = number
```

Comment:
hmmm -- do we expect this test to output nothing?

Reply:
Nope, sorry I missed this when running my tests. It was due to using the source path object instead of the Walkdir entry, tested again and should be working as expected.